sql/moz-fx-data-shared-prod/mozilla_vpn_derived/login_flows_v1/init.sql

```diff
@@ -3,6 +3,8 @@ CREATE TABLE IF NOT EXISTS
     flow_id STRING,
     flow_started TIMESTAMP,
     flow_completed TIMESTAMP,
+    login_completed TIMESTAMP,
+    reg_completed TIMESTAMP,
```

Comment:
`registration` instead of `reg` maybe? Without context it could be mistaken for `regular`.